src/shared/libs/type-checkers.ts

```diff
@@ -29,6 +29,10 @@ export function isFalsy<T>(value: T | Falsy): value is Falsy {
   return !value || Number.isNaN(value as number);
 }
 
+export function isNull(value: unknown): value is null {
+  return value === null;
+}
```

Comment:
why do we need this function when we have `value === null` and `value !== null`?

Reply:
The `isNull` function provides benefits like:

1. **Readability:** It makes the intent clearer and the code easier to understand.
2. **Reusability:** Centralizing null-check logic keeps the code DRY and easier to maintain.
3. **Type Safety:** It acts as a TypeScript type guard, improving type narrowing and ensuring safer code.